### ISSUE TITLE
wallabag can't retrieve contents for this article. Please report this issue to us.…

### ISSUE THREAD

**Issue by littleyoda** [wallabag/wallabag#2189]
Issue details

wallabag can't retrieve contents for this article. Please report this issue to us.…

Link: http://www.arduino-tutorial.de/2014/01/ein-altes-handy-als-arduino-input/

```
wallabag version (or git revision) that exhibits the issue: 2.0.5
How did you install wallabag? git clone
Last wallabag version that did not exhibit the issue (if applicable):
php version: PHP 5.5.9-1ubuntu4.17 (cli)
OS: Ubuntu 14.04 TLS
type of hosting (shared or dedicated): dedicated
which storage system you choose at install (SQLite, MySQL/MariaDB or PostgreSQL): sqlite
```

**Comment by techexo** [comment]
Was fetching fine with 2.2.3, I just made a PR to delete duplicate code snippets due to Crayon Syntax Highlighter.

**Comment by Kdecherf** [comment]
The referenced PR has been merged into graby-site-config 1.0.68 which was released along with wallabag 2.3.4, I'm closing this issue.
